src/storage/mod.rs

```diff
@@ -1033,6 +1033,7 @@ impl<E: Engine, L: LockManager> Storage<E, L> {
                                 key,
                                 start_ts,
                                 &bypass_locks,
+                                ctx.get_isolation_level(),
```

Comment:
I'm afraid there might be many code like `if ctx.get_isolation_level() == IsolationLevel::xxx` that needs to be updated bot missed. Does line 1027 needs to be updated?

Reply:
Yes, thankds for reminding. The entry of `check_ts_conflict` is unified as two more isolation variables `RC` and `RcCheckTS` are introduced for different purposes.

Reply:
`need_check_locks_in_replica_read` needs to return true when the isolation level is `IsolationLevel::RcCheckTs`.

Reply:
The isolation level filed is not passed by the read index request yet and the `RcCheckTS` read is not compatibile with replica read. Maybe we just document about it and solve the compatible issue next?

Reply:
I'll add the check in the tidb side so these two would not be used together.Maybe it's not necessary to support it.